backend/core/views.py

```diff
@@ -106,19 +107,17 @@ def create(self, request):
         return Response({"detail": "Logged out successfully."}, status=200)
 
 
-# Activate account by email
-def activate(request, uidb64, token):
-    User = get_user_model()
-    try:
-        uid = force_str(urlsafe_base64_decode(uidb64))
-        user = User.objects.get(pk=uid)
-    except (TypeError, ValueError, OverflowError, User.DoesNotExist):
-        user = None
-    if user is not None and account_activation_token.check_token(user, token):
-        user.is_active = True
-        user.save()
-        return HttpResponse(
-            "Thank you for your email confirmation. Now you can login your account."
-        )
-    else:
-        return HttpResponse("Activation link is invalid.")
+# Activates user. https://protocolostomy.com/2021/05/06/user-activation-with-django-and-djoser/
```

Comment:
"Djoser does not have a back end view to handle the front end URL that is the default ACTIVATION_URL."

This workaround is totally valid for backend function testing.
[Here's how we can restrict it to debug mode only](https://stackoverflow.com/questions/65241753/add-view-to-django-only-for-testing-not-in-production)
That should still be implemented in this PR.